### PR TITLE
Enable json api next episode switch

### PR DESCRIPTION
**Pull request by davidthomas22**
### Context

> Does this issue have a Trello card?

https://trello.com/c/1uu7BDVA

### Intent

> What changes are introduced by this PR that correspond to the above card?

To introduce JSON API next episode functionality

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/services/cms.js [hunk 1]
```diff
@@ -56,78 +59,77 @@ class CmsService {
...
+    );
     return homepages[0];
   }
+
+  async getNextEpisode(establishmentName, seriesId, seriesSortValue) {
+    const nextEpisodes = await this.#cmsApi.get(
+      new NextEpisodeQuery(
+        establishmentName,
+        seriesId,
+        seriesSortValue || undefined,
```
Comment (davidthomas22) [inline server/services/cms.js:128]: Passed in `seriesSortValue` or undefined as the below filter in the query did not accept a `null` value when a next episode is not passed. 

`.addFilter('series_sort_value', seriesSortValue, '>')`

File: server/services/__tests__/cmsService.spec.js [hunk 2]
```diff
@@ -254,17 +262,6 @@ describe('cms Service', () => {
...
         title: 'Buddhist reflection: 29 July',
       });
-
-      expect(contentRepository.nextEpisodesFor).toHaveBeenCalledWith({
-        episodeId: 1036,
-        establishmentId: 793,
-        id: 923,
-        perPage: 3,
-      });
-      expect(contentRepository.suggestedContentFor).toHaveBeenCalledWith({
```
Comment (andrewrlee) [inline server/services/__tests__/cmsService.spec.js:264]: shouldn't this still be called?

File: server/services/__tests__/cmsService.spec.js [hunk 3]
```diff
@@ -191,21 +207,12 @@ describe('cms Service', () => {
...
```
Comment (andrewrlee) [inline server/services/__tests__/cmsService.spec.js:201]: and here?

File: server/services/__tests__/cmsService.spec.js (hunk 2 above)
Comment (davidthomas22) [reply server/services/__tests__/cmsService.spec.js:264]: good spot

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**